charts/budibase/values.yaml

```diff
@@ -331,31 +344,65 @@ couchdb:
   ## `OrderedReady`
   podManagementPolicy: Parallel
 
+  ## To better tolerate Node failures, we can prevent Kubernetes scheduler from
```

Comment:
Given that a vast majority of users (I'd say a tentative 99%) won't touch almost any of the options from this point on, I'm wondering:

Should we continue to use the official couchDB helm chart at all? The couchDB team recommend that you don't cluster couch in K8S/docker - and even in our prod we use ansible provisioned EC2 instances. I'm thinking we may actually be better to only provide single node couchDB by default in our chart for simplicity, with the option to scale out the cluster if someone really desires it. Should we just hide all of the complex couch options in another values file, and show users how to pull them in or "merge" them into this chart if needed?

Reply:
I think we _should_ keep using the official CouchDB chart. It offers a lot of extras that users may want, such as monitoring through Prometheus, placement management, pod disruption budgets, etc. I've set the cluster size to 1 (away from the default of 3) to keep it in line with how the chart works currently. Clustering does work if you want to use it, and I suspect there will be customers out there that may want to.

If we want our values.yaml to be easier to navigate, we can remove anything that's a CouchDB default (which is most of this stuff) and point people over to their chart in our docs. I only put it in here for documentation purposes. It falls back to the default set in their values.yaml anyway, so removing them from here is a no-op.

Reply:
Makes sense - they do have a whole bunch of custom logic in there, and if it's simple enough for us to just drop in our image and keep using their chart then no point rewriting it. Let's just simplify the `values.yaml` here then and point to the official chart to make it easier for first timers and the standard use case!